src/coreclr/jit/emit.h

```diff
@@ -1978,6 +2007,17 @@ class emitter
         return (instrDescCGCA*)emitAllocAnyInstr(sizeof(instrDescCGCA), attr);
     }
 
+#ifdef FEATURE_LOOP_ALIGN
+    instrDescAlign* emitAllocInstrAlign()
+    {
+#if EMITTER_STATS
+        emitTotalIDescJmpCnt++;
```

Comment:
Seems odd to count this as a jump? Maybe introduce a new category?

Reply:
It was a copy-paste mistake. Should have been `emitTotalDescAlignCnt`.